docs/source/deduplication.rst

```diff
@@ -0,0 +1,40 @@
+DeDuplication of EBU-TT-Live documents
+======================================
+
+When documents are ReSequenced, duplication of ``style`` and ``region`` elements
+and attributes can occur. To address this, the DeDuplicator node takes in a
```

Comment:
No need to put input carriage mech details in here - manifest file only applies for filesystem input. Presumably deduplicator can be configured with any input carriage mechanism...

Reply:
Removed that bit of the line

Reply:
@SKHameed86 have you made any fixes here that you haven't yet pushed? I can't see any fix removing this bit...